spark/src/test/scala/org/apache/comet/parquet/ParquetReadSuite.scala

```diff
@@ -1125,7 +1123,9 @@ abstract class ParquetReadSuite extends CometTestBase {
   }
 
   test("row group skipping doesn't overflow when reading into larger type") {
-    assume(isSpark34Plus)
+    // Spark 4.0 no longer fails for widening types
+    // https://github.com/apache/spark/commit/3361f25dc0ff6e5233903c26ee105711b79ba967
+    assume(isSpark34Plus && !isSpark34Plus)
```

Comment:
Did you mean `assume(isSpark34Plus && !isSpark40Plus)` ?

Reply:
Yes, you are right. Thank you!